### DISCUSSION TITLE
More labels for github issues

### DISCUSSION THREAD

**Discussion by mcuee** [avrdudes/avrdude#1002] [General]
Just wondering what labels we should apply to issues (potentially bugs or invalid) before we confirm the status?

Eg: 
* https://github.com/avrdudes/avrdude/issues/509
* https://github.com/avrdudes/avrdude/issues/490

Shall we have a new label like "To be tested" or "to be worked on"? 

Then shall we have a few more labels?
1) General -- most of them can be moved to Dicussions
2) Support -- technical support which may be better served by avrchat
3) License -- license discussions (rare but sometimes happen)
4) OS specific labeld: Linux/macOS/Windows/BSD, issues which only affect one OS

**Comment by mcuee** [comment]
@dl8dtl @MCUdude @mariusgreuel 
BTW, I have tried to label the existing opening issues and pull requests, based on my understanding of the issues. Please change as you see fit. I will also try to label the closed pul requests and recent closed issues (say after 15 Dec 2021, not those on 14-Dec-2021).

**Comment by dl8dtl** [comment]
Within 20 years, there has never been a serious license issue (just a single, technical query), so I don't think we need a label for that.
An "unconfirmed" label might be good though.

**Comment by mcuee** [comment]
@dl8dtl I agree not to add `License` label. Please add the `unconfirmed` label. I can not add that. Thanks.

**Comment by mcuee** [comment]
Another one:
Misc fixes -- white space, spelling, etc (not so many here, so probably not that important).

**Comment by mcuee** [comment]
libusb has many labels but many of them are not relevant.
https://github.com/libusb/libusb/labels

DNM -- this may be good for discussions of pull-requests
build -- CMake or github action/CI
Wiki -- Wiki related
Release -- release related (this can be in Discussion)

**Comment by dl8dtl** [comment]
I added "unconfirmed" and "non-technical" as labels.

**Comment by mcuee** [comment]
Done the first cut labeling for the open issues and pull-requests, also done the labeling of closed pull-requests. Feel free to modify when you see fit.

**Comment by mcuee** [comment]
I have labeld the recent closed issues  after15 Dec 2021. I will continue to label the remaining 5 pages of unlabeled closed issues later.

**Reply by mcuee** [reply]
> I have labeld the recent closed issues after15 Dec 2021. I will continue to label the remaining 5 pages of unlabeled closed issues later.

Done for all the issues.

**Comment by mcuee** [comment]
This discussion can be closed.

**Comment by stefanrueger** [comment]
I was wondering whether  *non-functional* might make a good label. It's for those changes and requests that don't change the functionality of AVRDUDE, but perhaps increase maintainability, user experience, code logic or prepare for future extensions such as libavrdude etc. 

Maybe we could use *non-technical* for non-functional issues/PRs (I say so mainly because I don't know what that label is intended for).

**Comment by dl8dtl** [comment]
"non-functional" could be misleading, in the sense of "it doesn't function (work)".
I wonder whether we could find a clearer term for it, but otherwise, good idea.
